x-pack/plugins/entity_manager/server/lib/entities/built_in/hosts_from_ecs_data.ts

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EntityDefinition, entityDefinitionSchema } from '@kbn/entities-schema';
+import { BUILT_IN_ID_PREFIX } from './constants';
+
+export const builtInHostsFromEcsEntityDefinition: EntityDefinition = entityDefinitionSchema.parse({
+  id: `${BUILT_IN_ID_PREFIX}hosts_from_ecs_data`,
+  managed: true,
+  version: '1.0.0',
+  name: 'Hosts from ECS data',
+  description:
+    'This definition extracts host entities from common data streams by looking for the ECS field host.name',
+  type: 'host',
+  indexPatterns: ['filebeat-*', 'logs-*', 'metrics-*', 'metricbeat-*'],
+  identityFields: ['host.name'],
+  displayNameTemplate: '{{host.name}}',
+  history: {
```

Comment:
should we also align these settings with the services builtin https://github.com/elastic/kibana/blob/main/x-pack/plugins/entity_manager/server/lib/entities/built_in/services.ts#L36-L44